helm/flink-kubernetes-operator/values.yaml

```diff
@@ -190,6 +195,7 @@ fullnameOverride: ""
 jvmArgs:
   webhook: ""
   operator: ""
+  operatorLog: "-Dlog4j.configurationFile=/opt/flink/conf/log4j-operator.properties"
```

Comment:
Could the log args be simply part of jvmArgs.operator? (Although that may be a breaking change)

Reply:
entry point simply combines them for both operator and webhook: `$LOG_CONFIG $JVM_ARGS`

however moving the log default to jvmArgs.operator and jvmArgs.webhook would introduce duplication and not be compatible with existing usage of those settings (log settings would be lost for everyone that uses jvmArgs currently.